Build failed:

> ./magentoerpconnect/magento_model.py:276:8: E271 multiple spaces after keyword
> ./magentoerpconnect/magento_model.py:276:80: E501 line too long (80 > 79 characters)

--- a/magentoerpconnect/magento_model.py
+++ b/magentoerpconnect/magento_model.py
@@ -273,7 +273,7 @@ class magento_backend(orm.Model):
                                'import_products_from_date', context=context)
         return True
 
-    def  _domain_for_update_product_stock_qty(self, cr, uid, ids, context=None):
+    def _domain_for_update_product_stock_qty(self, cr, uid, ids, context=None):
         return [
             ('backend_id', 'in', ids),
             ('type', '!=', 'service'),

--- a/magentoerpconnect/magento_model.py
+++ b/magentoerpconnect/magento_model.py
@@ -283,7 +283,7 @@ class magento_backend(orm.Model):
         if not hasattr(ids, '__iter__'):
             ids = [ids]
         mag_product_obj = self.pool.get('magento.product.product')
-        domain = self._domain_for_update_product_stock_qty(self, cr, uid, ids,
+        domain = self._domain_for_update_product_stock_qty(cr, uid, ids,
                                                            context=context)
         product_ids = mag_product_obj.search(cr, uid, domain, context=context)
         mag_product_obj.recompute_magento_qty(cr, uid, product_ids,

